File: C++/A1/005.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main() {
    int m, d;
    cin >> m >> d;
    if ((m == 12 && d >= 21) || (m == 1) || (m == 2) || (m == 3 && d < 21))printf("winter\n");
    else if ((m == 3 && d >= 21) || (m == 4) || (m == 5) || (m == 6 && d < 21)) printf("spring\n");
    else if ((m == 6 && d >= 21) || (m == 7) || (m == 8) || (m == 9 && d < 21)) printf("summer\n");
    else printf("fall\n");
}

File: Python/A1/005.py
A = int(input())
B = int(input())
if(A >= 1 and A <= 3):
    if(A == 3 and B >= 21):
        print("spring")
    else:
        print("winter")
elif(A >= 4 and A <= 6):
    if(A == 6 and B >= 21):
        print("summer")
    else:
        print("spring")
elif(A >= 7 and A <= 9):
    if(A == 9 and B >= 21):
        print("fall")
    else:
        print("summer")
else:   
    if(A == 12 and B >= 21):
        print("winter")
    else:
        print("fall")